docs/cli-kit/ui-kit/contributing.md

```diff
@@ -0,0 +1,241 @@
+# Contributing to UI Kit
+
+## What is UI Kit
+
+UI Kit is a library that can be used to either ask for input from the user or render output to the terminal.
```

Comment:
Nitpick: Should we say that "UI Kit" are a set of components within CLI Kit?

Reply:
Also it's not just a set of component but also provides handy methods to help you use these components (`render...`)